control/control.CAASP.xml

```diff
@@ -362,6 +453,13 @@ configured accordingly to match the use case of the role. &lt;/p&gt;</label>
         <none_role_description>
           <label>• No services started by default.</label>
         </none_role_description>
+        <micro_os_role>
+          <!-- TRANSLATORS: a label for a system role -->
+          <label>SUSE MicroOS</label>
+        </micro_os_role>
+        <micro_os_role_description>
+          <label>• No services started by default.</label>
```

Comment:
I think this is not very helpful as it is identical description as MicroOS. So what is difference to Plain System role?